src/main/java/io/nitric/proto/event/v1/EventServiceGrpc.java

```diff
@@ -287,11 +287,11 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
   public static io.grpc.ServiceDescriptor getServiceDescriptor() {
     io.grpc.ServiceDescriptor result = serviceDescriptor;
     if (result == null) {
-      synchronized (EventGrpc.class) {
+      synchronized (EventServiceGrpc.class) {
         result = serviceDescriptor;
         if (result == null) {
           serviceDescriptor = result = io.grpc.ServiceDescriptor.newBuilder(SERVICE_NAME)
-              .setSchemaDescriptor(new EventFileDescriptorSupplier())
+              .setSchemaDescriptor(new EventServiceFileDescriptorSupplier())
               .addMethod(getPublishMethod())
               .build();
         }
```

Comment:
Is it feasible to also not version generated proto files to reduce noise as well? In some runtimes its not possible e.g. Go/PHP. But for runtimes that use packaging prior to deploy.

Reply:
I will look into it.